packages/atlas/src/joystream-lib/extrinsics.ts

```diff
@@ -746,4 +731,28 @@ export class JoystreamLibExtrinsics {
     const tx = await this.reactToVideoCommentTx(memberId, commentId, reactionId)
     return this.sendMetaprotocolMemberExtrinsic(tx, cb)
   }
+
+  async sendFundsTx(destinationAccount: MemberId, value: BN) {
+    await this.ensureApi()
+
+    return this.api.tx.balances.transfer(destinationAccount, value)
+  }
+
+  async sendFunds(destinationAccount: MemberId, value: BN, cb?: ExtrinsicStatusCallbackFn) {
```

Comment:
I didn't manage to transfer funds to another account. When I checked transactions in https://polkadot.js.org/apps the value is always 0. Something isn't working with passing BN here probably.